pkg/webui/console/components/events/details/index.js

```diff
@@ -39,9 +40,12 @@ const EventDetails = ({ className, children, event }) => {
 
   return (
     <div className={classnames(className, style.details)}>
-      {event.isSynthetic && <Notification content={messages.syntheticEvent} info small />}
       {!hasChildren ? (
-        <RawEventDetails className={style.codeEditor} details={event} id={getEventId(event)} />
+        event.isSynthetic ? (
+          <SyntheticEventDetails event={event} id={getEventId(event)} />
```

Comment:
`id` is not needed by `<SyntheticEventDetails />`

Reply:
nit; `<RawEventDetails />` expects the `details` prop, would be nice if `<SyntheticEventDetails />` follows this api as well.